src/composer.json

```diff
@@ -2,18 +2,18 @@
   "name": "packlink/prestashop",
   "description": "Packlink Shipping PrestaShop plugin",
   "type": "library",
-  "version": "2.2.7",
+  "version": "2.3.0",
   "repositories": [
     {
       "type": "vcs",
       "no-api": true,
-      "url": "git@github.com:packlink-dev/ecommerce_module_core.git"
+      "url": "git@github.com:logeecom/pl_module_core.git"
```

Comment:
The core should be updated, so you can now reference the pl core with version 3.0.1 (or the latest).

Reply:
This will not be done now, because one of the issues has been fixed in Core and the module needs to reference the ui-redesign branch in order to have those changes.